src/components/Popover/Popover.stories.tsx

```diff
@@ -49,6 +49,7 @@ export const Default: StoryObj<Args> = {
 
 export const Arrow: StoryObj<Args> = {
   args: {
+    // maybe gets shallow merged with the default args (?)
```

Comment:
I'm having trouble understanding the purpose of this comment. Was this a note to yourself?

Reply:
Yah, it's mostly a note that can be removed. @jinlee93 and I were pairing on this and he wasn't sure how the object gets used when consumed will the `args` merge or does the default one get overwritten?

Reply:
I believe the `args` are shallow merged, tested via 
![image](https://user-images.githubusercontent.com/86632227/203438393-5463d660-ad7f-4bac-a8d6-1eee3008e354.png)
Which results in `className: 'haw'` and `placement:'top-start'`